aws/acm_types.go

```diff
@@ -0,0 +1,36 @@
+package aws
+
+import (
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/go-commons/errors"
+)
+
+// ACMPA - represents all ACMPA
+type ACM struct {
+	ARNs []string
+}
+
+// ResourceName - the simple name of the aws resource
+func (acm ACM) ResourceName() string {
+	return "acm"
+}
+
+// ResourceIdentifiers - The volume ids of the ebs volumes
```

Comment:
is the resource identifiers referring to ebs volumes?

Reply:
thanks, fixed that comment